castervoice/lib/settings.py

```diff
@@ -496,9 +496,11 @@ def initialize():
         _USER_DIR = user_data_dir(appname="caster", appauthor=False)
     _SETTINGS_PATH = str(Path(_USER_DIR).joinpath("settings/settings.toml"))
 
-    for directory in ["data", "rules", "transformers", "hooks", "sikuli", "settings"]:
+    for directory in ["data", "sikuli", "settings"]:
```

Comment:
We might as well bundle the rest of this into update_user_dir_packages_to_v1_7_0. Then it can be to simply update user directory. Beyond version migration I suppose if someone chooses to do a custom user directory path this could be reused migrate the files.

Reply:
I agree. I'll move this stuff into the migrator util as well.